### PR TITLE
Fix dev dependecies to avoid issues for latest version

### LINKED ISSUE #120: Pin all dependencies in requirements-dev.txt.
### Environment
- `drf-user` version(s) (`drf-user --version`: 3.12.4
- Operating System(s): Mac
- Python version(s): 3.9.7

### Description of the bug
pin dependencies in requirements-dev.txt.

### What you expected to happen
pin dependencies, run commands and see everything runs fine.

### How to reproduce (as minimally and precisely as possible)
NA

### Anything else we need to know?
NA

### PR DESCRIPTION
**Pull request by inovizz**
## Description
Pin all dependencies in requirements-dev.txt

## Motivation and Context
pin dependencies, run commands and see everything runs fine.

## Have you tested this? If so, how?
Yes, it works.

image
image

## Checklist for PR author(s)

- [x] Changes are covered by unit tests (no major decrease in code coverage %).
- [x] All tests pass.
- [x] Docstring coverage is **100%** via `interrogate drf_user` (I mean, we _should_ set a good example :smile:).
- [x] Updates to documentation:
    - [ ] Document any relevant additions/changes in `README.md`.
    - [ ] Manually update **both** the `README.md` _and_ `docs/index.rst` for any new changes.
    - [ ] Any changed/added classes/methods/functions have appropriate `versionadded`, `versionchanged`, or `deprecated` [directives](http://www.sphinx-doc.org/en/stable/markup/para.html#directive-versionadded).  Find the appropriate next version in the project's [``__init__.py``](https://github.com/101loop/drf-user/blob/master/drf_user/__init__.py) file.

### Issue
Resolves #120

### CODE REVIEW

**Comment by sumit4613** [comment]
@inovizz Can you please pin dependencies in `requirements.txt` as well, if possible. Otherwise, I'll create a separate issue for that. 
BTW thanks for working on it. ❤️

**Comment by inovizz** [comment]
Yes, I'll work further on this in sometime.

File: requirements-dev.txt [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-model_bakery
-pre-commit
-pytest
-pytest-cov
-pytest-django
-sphinx
+black==21.9b0
+Faker==8.13.2
+flake8==3.9.2
+ipython==7.27.0
```
Comment (sumit4613) [inline requirements-dev.txt:5]: Looks like `ipython==7.27.0` is not supported for Python 3.6. That's why CI is failing for Python 3.6

Comment (inovizz) [reply requirements-dev.txt]: Sure, I've fixed this.

**Comment by inovizz** [comment]
> @inovizz Can you please pin dependencies in `requirements.txt` as well, if possible. Otherwise, I'll create a separate issue for that.
> BTW thanks for working on it. ❤️

Hi @sumit4613 - I think, I'll raise a separate PR to fix dependencies in the requirements.txt file.

Also, before that, I'll probably ask again that which DRF and Django version we plan to support? I propose Django 3.2.7 which is an LTS version and should be valid till 2024. DRF could be 3.12.x series version. What do you think?

cc @iamhssingh

**Review by sumit4613 — APPROVED** [review]
LGTM 💯 Thanks @inovizz for this.

**Pull request merged**